<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetectionSettings</class>
 <widget class="QDialog" name="DetectionSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detection Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Detectionmethod</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Input</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Blurred Orignial</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Difference (Original - Blurred)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Median filtered Difference</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Thresholded Difference</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Blurred Thresholded</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Detected</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="label_Image">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="checkBoxCrosshair">
     <property name="text">
      <string>draw crosshair</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox_Method">
     <item>
      <property name="text">
       <string>default Xray marker</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Blobdetection</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>white marker</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>corner detection</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>no detection</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>xray marker (center of mass)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>white Blobdetection</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Threshold offset in percent</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSpinBox" name="spinBox_ThresholdOffset">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
